tests/core/snap-set-core-config/task.yaml

```diff
@@ -131,9 +134,9 @@ execute: |
     sysctl -n kernel.printk|MATCH "^4\s+"
 
     echo "setting the timezone works"
-    tests.cleanup defer timedatectl set-timezone "$(cat /etc/timezone)"
+    tests.cleanup defer timedatectl set-timezone "$(timedatectl show -p Timezone --value)"
     snap set system system.timezone=Europe/Malta
-    MATCH "Europe/Malta" < /etc/timezone
+    timedatectl show -p Timezone --value | MATCH "Europe/Malta"
```

Comment:
It seems both `-p` and `--value` was introduced in 239, which seems to fail on multiple distros